vitrina/settings.py

```diff
@@ -38,6 +39,7 @@
 
 ALLOWED_HOSTS = ['localhost', '127.0.0.1', 'data.gov.lt', 'staging.data.gov.lt']
 
+TEST = any(['test' in arg for arg in sys.argv])
```

Comment:
Settings is not the place to recognize if we are running tests. Settings should be modified trough environment variables, using [django-environ](https://django-environ.readthedocs.io/).

```suggestion
```